actuator/src/main/java/org/tron/core/actuator/DelegateResourceActuator.java

```diff
@@ -158,7 +158,7 @@ public boolean validate() throws ContractValidateException {
         long accountNetUsage = ownerCapsule.getNetUsage();
         if (null != this.getTx() && this.getTx().isTransactionCreate()) {
           accountNetUsage += TransactionUtil.estimateConsumeBandWidthSize(dynamicStore,
-                  ownerCapsule.getBalance());
+                  ownerCapsule.getFrozenV2BalanceForBandwidth());
```

Comment:
Is there a possibility of data inconsistency due to this bug?

Reply:
Online has always been setBalance(ownerCapsule.getFrozenV2BalanceForBandwidth()); assignment, but the assignment is wrong when optimizing the code, this is to fix the problem in the code that has not been online